pandas/tests/indexes/test_base.py

```diff
@@ -2614,3 +2614,22 @@ def construct(dtype):
         no_matches = np.array([-1] * 6, dtype=np.intp)
         tm.assert_numpy_array_equal(result[0], no_matches)
         tm.assert_numpy_array_equal(result[1], no_matches)
+
+
+@pytest.mark.parametrize(
```

Comment:
this should go with other indexing tests e.g. pandas/tests/indexes/test_index.py.

can you add an object array that contains nans
can you add a DTI that has multiple NaTs

Reply:
I don't see a file called pandas/tests/indexes/test_index.py. Did you mean pandas/tests/indexes/test_indexing.py? In the file docstring, it mentions it contains tests for `get_indexer`.

Reply:
Moved and added tests